[libusb] x64-windows build failure "file COPY cannot read symlink"

**Host Environment**
- OS: [Microsoft Windows [Version 10.0.19041.388]]
- Compiler: 
   - Visual Studio 2017 Version: 15.9.25  VS Tools Version: 14.16.27023
   - Visual Studio 2019 Version: 16.6.5  VS Tools Version: 14.26.28801

**To Reproduce**
Steps to reproduce the behavior:
`set VCPKG_DEFAULT_TRIPLET=x64-windows`
`./vcpkg install libusb --editable `

**Failure logs** 
- Cut and past the appropriate build messages from the console output.
`->vcpkg install libusb --editable`
`Computing installation plan...`
`The following packages will be built and installed:`
`    libusb[core]:x64-windows`
`Detecting compiler hash for triplet x64-windows...`
`Starting package 1/1: libusb:x64-windows`
`Building package libusb[core]:x64-windows...`
`-- Using cached D:/Git/vcpkg.downloads/libusb-libusb-e782eeb2514266f6738e242cdcb18e3ae1ed06fa.tar.gz`
`-- Using source at D:/Git/vs2017.prod/buildtrees/libusb/src/3ae1ed06fa-f8218372d8`
`-- Building msvc/libusb_dll_2017.vcxproj for Release`
`CMake Error at scripts/cmake/vcpkg_install_msbuild.cmake:160 (file):`
**`  file COPY cannot read symlink`**
**`  "D:/Git/vs2017.prod/buildtrees/libusb/src/3ae1ed06fa-f8218372d8/README" to
  duplicate at`**
**`  "D:/Git/vs2017.prod/buildtrees/libusb/x64-windows-rel/3ae1ed06fa-f8218372d8/README":`**
`  No such file or directory.`
`Call Stack (most recent call first):`
`  ports/libusb/portfile.cmake:42 (vcpkg_install_msbuild)`
`  scripts/ports.cmake:79 (include)`

`Error: Building package libusb:x64-windows failed with: BUILD_FAILED`
`Please ensure you're using the latest portfiles with `.\vcpkg update`, then`
`submit an issue at https://github.com/Microsoft/vcpkg/issues including:`
`  Package: libusb:x64-windows`
`  Vcpkg version: 2020.06.15-nohash`

`Additionally, attach any relevant sections from the log files above.`

- Please attach any additional failure logs mentioned in the console output.
  No Logs created

**Additional context**
1. `->dir README*` shows the file README is a symlink to the actual file README.md
` Directory of D:\Git\vs2017.prod\buildtrees\libusb\src\3ae1ed06fa-f8218372d8`

**`07/29/2020  18:22    <SYMLINK>                   BUILTIN\Administrators README [README.md]`**
`08/28/2019  16:56             1,854              BUILTIN\Administrators README.git`
`08/28/2019  16:56             1,328              BUILTIN\Administrators README.md`

2. I researched and changed the `<vcpkg-root>\ports\libusb\portfile.cmake`, adding a line before line 42 to delete the symlink then the port will install properly.
   - From:
`39      endif()`
`40  endif()`
`41`
`42  vcpkg_install_msbuild(`
`43      SOURCE_PATH ${SOURCE_PATH}`

   - To:
`39      endif()`
`40  endif()`
`41`
`42  file(REMOVE ${SOURCE_PATH}/README)`
`43`
`44  vcpkg_install_msbuild(`
`45      SOURCE_PATH ${SOURCE_PATH}`

3. Please let me know if you wish me to submit a PR to make this change to master?

Thank you and have a wonderful day :)

--- a/ports/libusb/CONTROL
+++ b/ports/libusb/CONTROL
@@ -1,5 +1,6 @@
 Source: libusb
-Version: 1.0.23-1
+Version: 1.0.23
+Port-Version: 2
 Homepage: https://github.com/libusb/libusb
 Description: a cross-platform library to access USB devices
 Supports: !uwp

--- a/ports/libusb/portfile.cmake
+++ b/ports/libusb/portfile.cmake
@@ -39,6 +39,9 @@ if(VCPKG_TARGET_IS_WINDOWS)
       endif()
   endif()
 
+  # The README file in the archive is a symlink to README.md and cause issues with the windows MSBUILD process
+  file(REMOVE ${SOURCE_PATH}/README)
+
   vcpkg_install_msbuild(
       SOURCE_PATH ${SOURCE_PATH}
       PROJECT_SUBPATH msvc/libusb_${LIBUSB_PROJECT_TYPE}_${MSVS_VERSION}.vcxproj

--- a/ports/libusb/portfile.cmake
+++ b/ports/libusb/portfile.cmake
@@ -39,7 +39,8 @@ if(VCPKG_TARGET_IS_WINDOWS)
       endif()
   endif()
 
-  # The README file in the archive is a symlink to README.md and cause issues with the windows MSBUILD process
+  # The README file in the archive is a symlink to README.md 
+  # which causes issues with the windows MSBUILD process
   file(REMOVE ${SOURCE_PATH}/README)
 
   vcpkg_install_msbuild(

